grafonnext/schemas/latest/composable/panel/dashlist.json

```diff
@@ -0,0 +1,79 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "dashlist",
+    "version": "0.0"
+  },
+  "paths": {},
+  "components": {
+    "schemas": {
+      "dashlist": {
+        "type": "object",
+        "required": [
+          "PanelLayout",
+          "PanelOptions"
```

Comment:
Panel composable kinds are what i'm tentatively calling a grouped lineage - see grafana/thema#62. There isn't official support for these in Thema yet. Consequently, these are generated as a single schema object, rather than the (maybe, depending on how you plan to use these) preferable approach of generating them as two distinct schemas.

My guess is that, at least for now, you can hardcode what you need in jsonnet

Reply:
It looks like `dashlist.PanelLayout` and `dashlist.PanelOptions` needs to be a seperate schema under `components.schemas` instead of a property of the `dashlist` schema. The ref further down already seems to know this:
```
"layout": {
    "$ref": "#/components/schemas/dashlist.PanelLayout"
 },
```